docs/02_database/05_database-interface/01_entitydb.mdx

```diff
@@ -204,9 +203,9 @@ val recordB = list[1]
 ### getBulk
 
 This operation creates a `Flowable` of the whole table. If the database layer supports it, the records will be
-sorted in ascending order by the index provided, or by the primary key if none is provided. 
+sorted in ascending order by the index provided, or by the primary key if none is provided.
 
-Currently, only FoundationDb and Postgres support an ordered `getBulk`. There is also the `getBulkFromEnd` function, which returns records in descending order. 
+Currently, only FoundationDb and Postgres support an ordered `getBulk`. There is also the `getBulkFromEnd` function, which returns records in descending order.
```

Comment:
is this true? I thought all SQL layers would behave the same way?

Reply:
This is true, probably a left over from when we had fdb aerospike and postgres support only